cli/main.go

```diff
@@ -10,10 +12,19 @@ import (
 
 const sentryFlushTimeout = 5 * time.Second
 
+//nolint:gocritic
 func main() {
+	defer func() {
+		err := recover()
+		if err != nil {
+			originalMessage := fmt.Sprintf("panic: %v\n\n%s", err, string(debug.Stack()))
+			sentry.CurrentHub().CaptureMessage(originalMessage)
+			sentry.Flush(sentryFlushTimeout)
+			panic(err)
+		}
+	}()
+
 	if err := cmd.NewCmdRoot().Execute(); err != nil {
-		sentry.CaptureMessage(err.Error())
-		sentry.Flush(sentryFlushTimeout)
 		os.Exit(1)
 	}
 	sentry.Flush(sentryFlushTimeout)
```

Comment:
No need for this line any more, and the const can also be removed.

Reply:
It is actually needed becayse call to `CaptureMessage` are not sent synchronously by default , but I changed the transport to sync now so we can avoid it as we call sentry only in one place

Reply:
We still don't this line because if it panics and we're sending a sentry report in the defer, this line won't run anyway.

Reply:
Oh I see. you commented on another line. sorry.